libnd4j/include/legacy/cpu/NativeOpExecutioner.cpp

```diff
@@ -829,13 +829,14 @@ void NativeOpExecutioner::execScalar(sd::LaunchContext *lc, int opNum, const voi
   auto xType = sd::ArrayOptions::dataType(hXShapeInfo);
   auto yType = sd::ArrayOptions::dataType(hScalarShapeInfo);
   auto zType = sd::ArrayOptions::dataType(hZShapeInfo);
-
   if (shape::isEmpty(hXShapeInfo) || shape::isEmpty(hScalarShapeInfo)) return;
 
 #ifdef SD_EXPERIMENTAL_ENABLED
   BUILD_PAIRWISE_SELECTOR(xType, yType, zType, functions::scalar::ScalarTransform,
                           ::transform(opNum, hX, hXShapeInfo, hZ, hZShapeInfo, hScalar, extraParams), SD_COMMON_TYPES,
                           SD_COMMON_TYPES_ALL);
+  sd_printf("execScalar: RAfter unning experimental transform\n",0);
```

Comment:
Should this really be an `sd_printf`? Even if someone builds with the experimental flag, I'd assume that this would be meant to be a debug print?

Reply:
Fixed. The editor doesn't highlight parts that don't show up in the ifdef..just missed that one. Thanks for highlighting!